config/locales/state.en.yml

```diff
@@ -5,7 +5,7 @@ en:
     awaiting_responder_email: Waiting to be accepted
     awaiting_dispatch: Ready to send
     drafting: Draft in progress
-    pending_dacu_clearance: Pending clearance
+    pending_dacu_clearance: Pending clearance - Disclosure
```

Comment:
Do we want to change this? It will change it on all the case listings

Reply:
Reverted so available on rebase to master